Module/Scheduler/DB/pkg_SchedulerLoad.pkb

```diff
@@ -3702,6 +3723,61 @@ exception when others then
   );
 end deleteBatch;
 
+/* pproc deleteModuleBatch
+  �������� ���� ������, ������������� ������
+  ������������ ��� �������� ������, ������������� ������ ��� ��� �������������
+  
+  ���������:
+  moduleName                  - ������������ ������
+*/
+procedure deleteModuleBatch(
+  moduleName varchar2 
+)
+is
+  -- id ������
+  moduleId integer;
+begin
+  moduleId := pkg_ModuleInfo.getModuleId(
+    moduleName            => moduleName
+    , raiseExceptionFlag  => 1
+  );
+  for curBatch in (
+select
+  b.batch_id
+  , b.batch_short_name
+  , b.activated_flag
+from
+  sch_batch b
+where
+  b.module_id = moduleId
+  )
+  loop
+    if curBatch.activated_flag = 1 then
+      pkg_Scheduler.deactivateBatch(
+        batchId      => curBatch.Batch_Id
+        , operatorId => pkg_operator.getCurrentUserId()
+      );
+    end if;
+    deleteBatch(
+      batchId         => curBatch.batch_id
+    );
+    dbms_output.put_line(

```

Comment:
1. use outputInfo
2. use one language not mixture of English and Russian
3. use the message in deleteBatch(batchId=>) as suggested earlier